README.md

```diff
@@ -35,10 +35,12 @@ sudo snap install rt-tests
 
 ## Configure
 
-It's necessary to connect the [process-control](https://snapcraft.io/docs/process-control-interface) interface to work properly:
-
+It's necessary to connect the [process-control](https://snapcraft.io/docs/process-control-interface), [mount-observe](https://snapcraft.io/docs/mount-observe-interface), [system-trace](https://snapcraft.io/docs/system-trace-interface) and connect the `custom-cpu-latency` plug into the `custom-cpu-latency-dev` slot to work properly:
```

Comment:
Too many of them in a single sentence, not telling which interface is needed for what purpose. Will cyclictest also not work properly? And what does it mean to not work "properly"? Will it be slow, nonfunctional, unpredictable??

Let's refactor these in another PR, along with the other interface documentations.